tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -1204,6 +1207,9 @@ create_compressed_scan_paths(PlannerInfo *root, RelOptInfo *compressed_rel, int
 		add_partial_path(compressed_rel, compressed_path);
 	}
 
+	bool old_bitmapscan = enable_bitmapscan;
+	enable_bitmapscan = false;
+
```

Comment:
```suggestion
	  /*
   * We set enable_bitmapscan to false here to ensure any pathes with bitmapscan do not
   * displace other pathes. Note that setting the postgres GUC will not actually disable
   * the bitmapscan path creation but will instead create them with very high cost.
   * If bitmapscan were the dominant path after postgres planning we could end up
   * in a situation where we have no valid plan for this relation because we remove
   * bitmapscan pathes from the pathlist.
  */ 
	bool old_bitmapscan = enable_bitmapscan;
	enable_bitmapscan = false;
```

Reply:
Updated.